Gems/ROS2/gem.json

```
@@ -1,5 +1,9 @@
 {
     "gem_name": "ROS2",
+    "version": "1.0.0",
+    "platform": [
```

The field name is actually plural, and I just noticed it is missing from the docs - will update my doc PR
```suggestion
    "platforms": [
```